src/course-page/CoursePage.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@edx/paragon';
+
+function CoursePage({
+  title,
+  description,
+  configure,
+  coursePageStatus,
+  enabledButton,
+}) {
+  const getContainerClasses = () => {
+    let classes = 'bg-white shadow p-3 m-3';
+    classes += coursePageStatus ? ' border-enabled' : ' border-disabled';
+    return classes;
+  };
```

Comment:
Instead of this, use the `classNames` utility. You can read more about that [here](https://www.npmjs.com/package/classnames), but to give a brief intro with that you can replace the above with:

```js
classNames('course-page-tile bg-white shadow p-3 m-d', {'disabled': !coursePageStatus});
```